tests/plugins/balloontoolbar/context/focusablebuttons.js

```diff
@@ -0,0 +1,87 @@
+/* bender-tags: balloontoolbar */
+/* bender-ckeditor-plugins: balloontoolbar, basicstyles, button, toolbar */
+
+( function() {
+	'use strict';
+
+	bender.editor = {
+		startupData: '<p>Hello <strong>world</strong>.</p>',
+		config: {
+			extraAllowedContent: 'strong em'
+		}
+	};
+
+	bender.test( {
+		'test check if balloon toolbar elements are registered as focusable': function() {
```

Comment:
This TC name could be simplified to "test if balloon toolbar elements are registered as focusable".